app/assets/javascripts/discourse/components/topic-list-item.js.es6

```diff
@@ -64,6 +75,11 @@ export const ListItemDefaults = {
     }
   },
 
+  @observes("topic.pinned")
```

Comment:
Also if you can't remove it, if it's only doing this, you can directly put it on top of renderTopicListItem, it's not because a function is decorated that you can't call it.

Reply:
yep it is still needed